openrpc.json

```diff
@@ -0,0 +1,1803 @@
+{
```

Comment:
what is this file is this a test openrpc.json if so we should probably re-label it as so or create a fixtures directory for it.

Reply:
oops! deleting